### PR TITLE
Bg fix first part

### PR DESCRIPTION
**Pull request by sfranky**

### CODE REVIEW

File: qtop.py [hunk 1]
```diff
@@ -195,10 +196,11 @@ def nodes_with_jobs(worker_nodes):
 def display_job_accounting_summary(cluster_dict, total_running_jobs, total_queued_jobs, qstatq_list):
     if options.REMAP:
         print '=== WARNING: --- Remapping WN names and retrying heuristics... good luck with this... ---'
-    print '\nPBS report tool. Please try: watch -d ' + QTOPPATH + \
-          '. All bugs added by sfranky@gmail.com. Cross fingers now...\n'
-    print colorize('===> ', '#') + colorize('Job accounting summary', 'Nothing') + colorize(' <=== ', '#') + colorize(
-        '(Rev: 3000 $) %s WORKDIR = to be added', 'account_not_coloured') % (datetime.datetime.today())
+    print 'PBS report tool. All bugs added by sfranky@gmail.com. Cross fingers now...'
```
Comment (fgeorgatos) [inline qtop.py]: LOL ;-)

File: qtop.py [hunk 2]
```diff
@@ -294,6 +297,15 @@ def create_job_counts(user_names, job_states, state_abbrevs):
...
         job_counts['waiting_of_user'].setdefault(user_name, 0)
         job_counts['exiting_of_user'].setdefault(user_name, 0)
+        job_counts['restarting_of_user'].setdefault(user_name, 0)
+        job_counts['Eqw_of_user'].setdefault(user_name, 0)
+        job_counts['hold_of_user'].setdefault(user_name, 0)
+        job_counts['transferring_of_user'].setdefault(user_name, 0)
+        job_counts['threshold_reached'].setdefault(user_name, 0)
+        job_counts['job_ending_of_user'].setdefault(user_name, 0)
+        job_counts['suspended_of_user'].setdefault(user_name, 0)
+        job_counts['suspended_by_the_queue'].setdefault(user_name, 0)
```
Comment (fgeorgatos) [inline qtop.py]: a little bit too many states and they are all hardwired here, which is not too great (as said, to be done later)

File: qtop.py [hunk 3]
```diff
@@ -703,7 +715,24 @@ def calculate_wn_occupancy(cluster_dict, user_names, job_states, job_ids):
...
-        print core_line
+        try:
+            print core_line
+        except IOError:
+            # This tries to handle the broken pipe exception that occurs when doing "| head"
+            # stdout is closed, no point in continuing
+            # Attempt to close them explicitly to prevent cleanup problems
+            # Results are not always best. misbehaviour with watch -d,
+            # output gets corrupted in the terminal afterwards without watch.
+            # TODO Find fix.
```
Comment (fgeorgatos) [inline qtop.py]: OK

File: qtop.py [hunk 4]
```diff
@@ -900,7 +932,12 @@ def exec_func_tuples(func_tuples):
 
     cwd = os.getcwd()
     QTOPPATH = os.path.expanduser(cwd)
-    config = load_yaml_config(QTOPPATH)
+    USERPATH = os.path.expandvars('$HOME/.local/qtop')
+    try:
+        config = load_yaml_config(USERPATH)
+    except IOError:
+        config = load_yaml_config(QTOPPATH)
```
Comment (fgeorgatos) [inline qtop.py]: for the record: this configuration is only an if-else situation; original qtop could "merge" the config vars across multiple configs; this is possibly a feature request

File: qtopconf.yaml [hunk 5]
```diff
@@ -129,9 +139,9 @@ filtering:
 
 # what parts of qtop to display
 user_display_parts:
-# - job_accounting_summary
+ - job_accounting_summary
  - workernodes_matrix
-# - user_accounts_pool_mappings
+ - user_accounts_pool_mappings
```
Comment (fgeorgatos) [inline qtopconf.yaml]: next feature request: 
* support reading `getent passwd` output (cached in a file) to show info per user in the right area

File: sge.py [hunk 6]
```diff
@@ -65,3 +65,41 @@ def make_stat(fn, write_method):
...
+        d['run'] = len(queue_elem.findall('./job_list[@state="running"]'))
+        d['lm'] = 0
+        d['queued'] = 0
+        qstatq_list.append(d)
+
+    total_running_jobs = str(sum([d['run'] for d in qstatq_list]))
+    for d in qstatq_list:
+        d['run'] = str(d['run'])
+        d['queued'] = str(d['queued'])
+    total_queued_jobs = str(len(root.findall('.//job_list[@state="pending"]')))
```
Comment (fgeorgatos) [inline sge.py]: hardwired? => feature request to un-hardwire

**Comment by fgeorgatos** [comment]
OK, LGTM, merging in.

File: qtop.py (hunk 2 above)
Comment (sfranky) [inline qtop.py]: FIXED in branch ch-stop-relying-on-pyyaml-37

File: qtopconf.yaml (hunk 5 above)
Comment (sfranky) [inline qtopconf.yaml]: added in issues

**Pull request merged**